### PR TITLE
feat: ani-sync -s arg

### PR DESCRIPTION
**Pull request by iamchokerman**
#688

### CODE REVIEW

**Comment by port19x** [comment]
I'll give it some thorough testing  with a friend

**Comment by port19x** [comment]
Seems to work, no issues so far

File: ani-cli [hunk 1]
```diff
@@ -453,15 +453,17 @@ open_episode () {
...
-	set -- "$player_fn" "$video_url"
+trackma_title="$(printf '%s' "$anime_id" | tr '-' ' ' | awk '{for(i=1;i<=NF;i++){ $i=toupper(substr($i,1,1)) substr($i,2) }}1')"
 	case "$player_fn" in
 		vlc)
 			[ ! "$auto_play" -eq 0 ] && set -- "$@" "--play-and-exit"
-			set -- "$@" --http-referrer="$dpage_link"
+			set -- "$player_fn" "$video_url"  --http-referrer="$dpage_link"
+			;;
+		syncplay|/Applications/Syncplay.app/Contents/MacOS/syncplay)
+			nohup "$player_fn" "$video_url" -- --referrer="$dpage_link" --force-media-title="$trackma_title $episode" > /dev/null 2>&1 &
```
Comment (Derisis13) [inline ani-cli:463]: I'm actually a bit worried about this line's compatibility with auto-play mode (whenever you specify an episode range). It's fine to exclude that option from syncplay if it doesn't make much sense, what I'm worried about is the program ending up in an undefined state. I'd rather have it play from line 471 or 474

**Review by Derisis13 — CHANGES_REQUESTED** [review]

File: ani-cli (hunk 1 above)
Comment (iamchokerman) [reply ani-cli:463]: at the moment it's not compatible with auto-play mode, but ideally i think it should be imo. i'll look into this tomorrow, ty for the suggestion

Comment (iamchokerman) [reply ani-cli]: nvm, it was an easy fix, i just decided to use both my eyes and brain at the same time for once

Comment (iamchokerman) [reply ani-cli]: everything working as expected now, don't know why i complicated things in the beginning, sorry for that

**Review by Derisis13 — APPROVED** [review]
Now it passes

**Review by port19x — APPROVED** [review]
Confirmed working

**Pull request merged**